src/scripts/farming/mutation/Mutation.ts

```diff
@@ -73,6 +73,14 @@ abstract class Mutation implements Saveable {
         return '';
     }
 
+    /**
+     * Handles getting the mutation chance
+     * @param idx The plot index
+     */
+    mutationChance(idx: number): number {
+        return this._mutationChance;
+    }
```

Comment:
This doesn't seem to be doing anything different than the old variable?
`idk` is never used either.

Reply:
Adding it so it can be overriden in the GrowNearFlavorMutation class.